src/main/scala/com/github/unchama/seichiassist/subsystems/sharedinventory/infrastracture/JdbcSharedInventoryPersistence.scala

```diff
@@ -0,0 +1,57 @@
+package com.github.unchama.seichiassist.subsystems.sharedinventory.infrastracture
+
+import cats.effect.Sync
+import com.github.unchama.seichiassist.subsystems.sharedinventory.domain.SharedInventoryPersistence
+import com.github.unchama.seichiassist.subsystems.sharedinventory.domain.bukkit.InventoryContents
+import com.github.unchama.seichiassist.util.ItemListSerialization
+import scalikejdbc.{DB, scalikejdbcSQLInterpolationImplicitDef}
+
+import java.util.UUID
+import scala.jdk.CollectionConverters._
+
+class JdbcSharedInventoryPersistence[F[_]: Sync] extends SharedInventoryPersistence[F] {
+
+  /**
+   * セーブされている[[InventoryContents]]を完全に削除します。
+   */
+  override def clear(targetUuid: UUID): F[Unit] = Sync[F].delay {
+    DB.localTx { implicit session =>
+      sql"UPDATE playerdata SET shareinv = NULL WHERE uuid = '${targetUuid.toString}'"
```

Comment:
```suggestion
      sql"UPDATE playerdata SET shareinv = NULL WHERE uuid = ${targetUuid.toString}"
```

scalikejdbcが自動で処理するのでクォートは要りません (以下同様)